src/btree/bt_read.c

```diff
@@ -184,6 +184,114 @@ __bt_reconstruct_delta(WT_SESSION_IMPL *session, WT_REF *ref, WT_ITEM *delta)
     return (ret);
 }
 
+/*
+ * __bt_unpacked_delta_cmp --
```

Comment:
This is to compare the key of the delta?

Reply:
yes

Reply:
Updated in [e013a9b](https://github.com/wiredtiger/wiredtiger/pull/11627/commits/e013a9bee2958e7f40c5f4c6894fa2c75528c693)